### DISCUSSION TITLE
Look up dangerous goods (hazmat) classification by ASIN

### DISCUSSION THREAD

**Discussion by spinneret** [amzn/selling-partner-api-models#3927] [General]
### Discussed in https://github.com/amzn/selling-partner-api-docs/discussions/3022

<div type='discussions-op-text'>

<sup>Originally posted by **spinneret** March 24, 2023</sup>
Is it possible to lookup if a product is classified as dangerous goods (hazmat) by ASIN? I reviewed the API reference in detail and couldn't find an endpoint that can be used to retrieve that information. </div>

**Comment by XDapps** [comment]
I'm having this same issue. Assuming your seller account is NOT approved for HazMat program. The Inbound Eligibility API will give you a "False" when checking inbound eligibility and will return a reason code. This reason code can be used to determine if the product is Dangerous Goods. 

The problem is, if the seller is approved for the Hazmat program, this query returns true and does not provide any reason code.

The only work around is to create an empty seller account that is not approved for Hazmat and use those seller credentials for the API calls to check Hazmat status.

I cannot believe this info is not directly available via API! (maybe it is, but I cannot find it).

**Reply by wlggraham** [reply]
There are some asins that are hazmat but don't require the seller to be part of the Hazmat program. We were following this same logic above for classifying hazmat asins and ran into this issue.
